packages/shared/src/components/post/PostContentContainer.tsx

```diff
@@ -21,10 +27,29 @@ function PostContentContainer({
   hasNavigation,
   className,
   children,
+  navigationProps,
+  isNavigationOutside,
 }: PostContentContainerProps): ReactElement {
+  const isMobile = useViewSize(ViewSize.MobileL);
+  const props = !isMobile && navigationProps;
   const Wrapper = hasNavigation ? BodyContainer : PageBodyContainer;
 
-  return <Wrapper className={className}>{children}</Wrapper>;
+  return (
+    <ConditionalWrapper
+      condition={isNavigationOutside}
+      wrapper={(component) => (
+        <>
+          {props && <FixedPostNavigation {...props} />}
+          {component}
+        </>
+      )}
+    >
+      <Wrapper className={className}>
+        {props && !isNavigationOutside && <FixedPostNavigation {...props} />}
```

Comment:
I don't think this double check is necessary since the wrapper is looking out for it `!isNavigationOutside`

Reply:
Yeah, but the wrapper renders this part too.